### ISSUE TITLE
Team Discussions

### ISSUE THREAD

**Issue by blitzmann** [pyfa-org/eos-old#15]
Tweetfleet is horrible for continued conversations due to it's message limit. We stuck with it simply because that's where everyone was gathered, but I initially preferred gitter.im for communications as it has unlimited messages.

However, I understand that no one wants to install even more mobile apps and have multiple chat tabs open, so... I'm proposing we create a new slack instance for pyfa communications. We will still have an upper limit of 10K messages, but it will take much much longer to reach that with a small team like ours. And I believe you have have multiple accounts signed into multiple slacks, so everything is contained.

The problem with slack, though, is that it's invite-only. That's a pretty major downside IMO, which is another reason I think gitter.im is a bit better. However, gitter.im doesn't have nearly the amount of integrations as Slack, and is relatively unknown.

There's also HipChat...

Regardless of what we choose, I don't think we can stay inside of tweetfleet. Thoughts?

**Comment by Regner** [comment]
I am totally open to gitter.im

**Comment by Ebag333** [comment]
There is one big advantage to staying where we are (outside of installing other clients, etc).

It's _super_ easy to pull people in as need be.  If we do a new slack team, for all intents and purposes it's completely separate, I'm already part of a few slack teams that are essentially dead because of the lack of participation in them.

Tweetfleet already has thousands of users on there, and trying to get people to setup a new slack team is less painful than trying to get them to install a whole new app, but it's still more painful than no extra work at all.

Basically just pointing out the cost associated with it.  I'll go wherever folks want to go.

If we end up going somewhere else, my vote would be for another Slack team.  It's fairly easy to add the team to the existing chat clients, and like you said Slack will have better integration than gitter.

**Comment by Regner** [comment]
That is one of the reasons why I would prefer Gitter. For the most part the people we want to pull into a conversation have a GitHub account. With that they can join us without having to create new accounts.

**Comment by blitzmann** [comment]
> That is one of the reasons why I would prefer Gitter. For the most part the people we want to pull into a conversation have a GitHub account. With that they can join us without having to create new accounts.

Pretty much my thought on it. The Tweetfleet channel is great and all because I can just mention someone and boom, invite. However, that's really the only thing great about it. It's overall extremely limiting. I would still like to have a pyfa channel that's maybe public on tweetfleet, and have a MOTD state if anyone is interested then head over to where ever we end up.

I also took a look at gitter integrations - much better than I originally thought. So there's some options.

**Comment by Regner** [comment]
There is one issue with gitter though, it doesn't like making channels for forks. So if we were to do that I would really like to see Eos move under the pyfa-org or the Eos repo in there made not a fork, or find a way to get around that.

**Comment by Ebag333** [comment]
Github doesn't even let you use their built in code search for forks.  :(

**Comment by Regner** [comment]
> Github doesn't even let you use their built in web based code search for forks. :(

Yea OK, even more reason to end the fork madness.

**Comment by blitzmann** [comment]
> web based code search
Wait, there's a code search on github? Please enlighten me to the ways!

**Comment by Ebag333** [comment]
I don't know how well it works, but it exists.  I tried searching for a few things on Pyfa (which isn't a fork) and couldn't find anything, but I spent about 5 seconds trying to make it work. vOv

Just pull the trigger and make it happen.  Since you're the owner of this @blitzmann you probably need to setup Gitter.  There doesn't need to be a presidential debate on it.  Dooooo itttttt.

**Comment by DarkFenX** [comment]
Let's keep this discussion in gitter / new slack.
